mgmt/rest/v2/config.go

```diff
@@ -65,7 +95,7 @@ func (s *apiV2) getPluginConfigItem(w http.ResponseWriter, r *http.Request, p ht
 	}
 
 	cdn := s.configManager.GetPluginConfigDataNode(typ, name, iver)
-	item := &PluginConfigItem{ConfigDataNode: cdn}
+	item := &PluginConfigItem{Config: cdn}
```

Comment:
As above, small suggestion: changing `cdn` to for example `cfg` 
Why: after changing the name ConfigDataNode to Config, `cdn` does not make sense as it had before.